desktop-environment.el

```diff
@@ -357,15 +365,21 @@ replacing the placeholder %d with the prefix argument."
 (defun desktop-environment-toggle-mute ()
   "Toggle between muted and un-muted."
   (interactive)
-  (message "%s"
-           (desktop-environment--shell-command-to-string desktop-environment-volume-toggle-command)))
+  (message "Volume %s"
```

Comment:
I'm wondering if "Sound [on]" would be better than "Volume [on]". What do you think?

Reply:
Yes, That's much better

Reply:
Yes, That's much better.